app/models/monthly_statistics/base.rb

```diff
@@ -19,7 +19,11 @@ def awaiting_decision_count(statuses)
     end
 
     def unsuccessful_count(statuses)
-      (statuses['declined'] || 0) + (statuses['rejected'] || 0) + (statuses['conditions_not_met'] || 0)
+      (statuses['declined'] || 0) +
+        (statuses['rejected'] || 0) +
+        (statuses['conditions_not_met'] || 0) +
+        (statuses['withdrawn'] || 0) +
+        (statuses['offer_withdrawn'] || 0)
```

Comment:
This makes sense to me. 

There is also a legacy `cancelled` status. There were 19 of these in the previous cycle but it doesn't appear in `status_before_deferral` so can be omitted.

Reply:
Great thanks for checking @stevehook